mica/stats/acq_stats.py

```diff
@@ -441,6 +447,50 @@ def _get_obsids_to_update(check_missing=False):
     return obsids
 
 
+def warn_on_acq_anom(acqs, emails):
+    """
+    Log and email warning about any acquisition stars with observed positions outside the
+    expected search box (plus a pad).  For acquisition stars with tracked positions in the wrong
+    search box, note the box (classic acquisition anomaly).
+
+    :param acqs: astropy table of acquisition stats, includes expected and observed position,
+                 obsid, slot, and halfw used in this method.
+    :param emails: list of addresses to receive email warning if any are generated
+    """
+    # Find tracked objects in the acq stats table outside the intended search box plus padding
+    # img_func == NONE means nothing was tracked
+    box_pad = 16  # arcsecs
+    anom_match = ((acqs['img_func'] != 'NONE') &
+                  ((np.abs(acqs['yang_obs'] - acqs['yang']) >= (acqs['halfw'] + box_pad))
+                   | (np.abs(acqs['zang_obs'] - acqs['zang']) >= (acqs['halfw'] + box_pad))))
+    for anom in acqs[anom_match]:
+        # Check to see if the star is actually found in another box.
+        other_box_match =  ((np.abs(anom['yang_obs'] - acqs['yang']) <= (acqs['halfw'] + box_pad))
+                            & (np.abs(anom['zang_obs'] - acqs['zang']) <= (acqs['halfw'] + box_pad)))
+        if np.any(other_box_match):
+            text = "Acquisition Anomaly.  Star for slot {} actually in box {} \n".format(
+                anom['slot'], acqs[other_box_match][0]['slot'])
+        else:
+            text = "Does not appear to be classic star-in-wrong-box anomaly\n"
+        # Make a dictionary of the anom record for use in string formatting
+        output_dict = {col: anom[col] for col in anom.colnames}
+        output_dict['dy'] = anom['yang_obs'] - anom['yang']
+        output_dict['dz'] = anom['zang_obs'] - anom['zang']
+        text += """Large Deviation from Expected ACQ Star Position in {obsid}
+      Slot {slot} Expected (Y-Pos, Z-Pos) = ({yang:.1f}, {zang:.1f})
+      Slot {slot} Observed (Y-Pos, Z-Pos) = ({yang_obs:.1f}, {zang_obs:.1f})
+      Halfwidth {halfw:03d}        (dy, dz) = ({dy:.1f}, {dz:.1f})""".format(
+            **output_dict)
+        # Log and Send message for slot.  Obsid can have more than one email
+        logger.warn(text)
+        msg = MIMEText(text)
+        msg['From'] = 'aca@head.cfa.harvard.edu'
```

Comment:
@taldcroft do you prefer these to come from the cron-task USER?  I note that's what you did in the kadi code most recently.